docs/reference/msbuild-targets.md

```diff
@@ -295,7 +295,7 @@ Additional restore settings may come from MSBuild properties in the project file
 
 | Property | Description |
 |--------|--------|
-| RestoreSources | Semicolon-delimited list of package sources. |
+| RestoreSources | Semicolon-delimited list of package sources. <br />When provided on the command line the semicolons may need to be replaced by escaped value %3B. <br />Due to the way the property is parsed, local folder sources must be in the uri format (i.e. prefixed with file:///) and may need to be placed after all internet sources. |
```

Comment:
I can't imagine why local sources would need to be after http sources, but I didn't try it out to confirm.

Reply:
msbuild escaping runs into weird problems.

Reply:
Sorry for the delay. The bug related to that is https://github.com/NuGet/Home/issues/4094 . I mentionned it in a commit, but not in the PR description.